docs/components.rst

```diff
@@ -0,0 +1,76 @@
+How to use components
+=====================
+
+
+So you've seen the new fancy buttons and want to give them a try, but dont know where to start. No problem!
+
+First lets cover the basics. Discord messages can have *components*, such as buttons, dropdowns etc. These components sit in whats called an "action row". An action row can hold 5 components at a time. Your message can have 5 action rows, so thats a total of 25 components!
+
+Sending some components
+_______________________
+
+First we need to create some buttons, lets put them in a list for now. We'll use :meth:`create_button() <discord_slash.utils.manage_components>` to create a green button
+
+.. code-block:: python
+
+    from discord_slash.utils import manage_components
+    from discord_slash.model import ButtonStyle
+
+    buttons = [
+                manage_components.create_button(
+                    style=ButtonStyle.green,
+                    label="A Green Button"
+                )
+              ]
+
+So we have a button, but where do we use it. Let's create an action row with :meth:`create_actionrow() <discord_slash.utils.manage_components>` and put our buttons in it
+
+.. code-block:: python
+
+    [...]
+    action_row = manage_components.create_actionrow(*buttons)
+
+
+Fantastic, we now have an action row with a green button in it, now lets get it sent in discord
+
+.. code-block:: python
+
+    await ctx.send("My Message", components=[action_row])
+
+Now if you've followed along, you have a green button in discord! But theres a problem, whenever you click it you see that the ``interaction failed``. Why is that?
+Well, in Discord, clicking buttons and using slash commands are called ``interactions``, and Discord doesn't know if we've received them or not unless we tell Discord. So how do we do that?
+
+Responding to interactions
+__________________________
+
+When responding to interactions, we have 2 choices, either to do it in the command itself, or a global event handler (like :meth:`on_slash_command_error`)
```

Comment:
I'm not sure about that wording, as it's choices of listening/waiting for interaction

Reply:
How about:

When responding, you have 2 choices in how you handle them. You can either wait for them in the command itself, or listen for them in a global event handler (like :meth:`on_slash_command_error`)

Reply:
When responding, you have 2 choices in how you handle interactions. You can either wait for them in the command itself, or listen for them in a global event handler (similar to :meth:on_slash_command_error)

Reply:
works for me, resolving